client.go

```diff
@@ -290,7 +333,7 @@ func getGMTTimestamp() string {
 		//
 		time = strings.Replace(time, "UTC", "GMT", -1)
 	}
-	return time
+	return timegit
```

Comment:
Yeah, fixing. I'll hold on the commit until we come to a conclusion about the parseError